contrib/migrate/migrate.c

```diff
@@ -0,0 +1,841 @@
+  ) DONT_REUSE_OID, COLLATE utf8_bin",
+
+  "create view [db_synonym] ( \
+  	[synonym_name] CHARACTER VARYING(255), \
+  	[synonym_owner_name] CHARACTER VARYING(255), \
+  	[is_public_synonym] CHARACTER VARYING(3), \
+  	[target_name] CHARACTER VARYING(255), \
+  	[target_owner_name] CHARACTER VARYING(255), \
+  	[comment] CHARACTER VARYING(2048) \
+  ) as \
+  SELECT \
+  	[s].[name] AS [synonym_name], \
+  	CAST ([s].[owner].[name] AS VARCHAR(255)) AS [synonym_owner_name], \
+  	CASE WHEN [s].[is_public] = 1 THEN 'YES' ELSE 'NO' END AS [is_public_synonym], \
+  	[s].[target_name] AS [target_name], \
+  	CAST ([s].[target_owner].[name] AS VARCHAR(255)) AS [target_owner_name], \
+  	[s].[comment] AS [comment] \
+	FROM \
+  [_db_synonym] [s] \
+  WHERE \
+    CURRENT_USER = 'DBA' \
+    OR [s].[is_public] = 1 \
+    OR ( \
+      [s].[is_public] = 0 \
+      AND { [s].[owner].[name] } SUBSETEQ ( \
+              SELECT SET {CURRENT_USER} + COALESCE (SUM (SET {[t].[g].[name]}), SET { }) \
+              FROM [db_user] [u], TABLE([groups]) AS [t]([g]) \
+              WHERE [u].[name] = CURRENT_USER \
+             ) \
+    )",
+
+  /* set system class for newly added tables and views */
+  "update _db_class set is_system_class = 1 where class_name in ('_db_server', 'db_server', '_db_synonym', 'db_synonym')"
+};
+
+static char *rename_query = "select \
+  	'rename table ' || class_name || ' to [' || lower (owner.name) || '.' || class_name || '] ' as q \
+   from  _db_class \
+   where is_system_class % 8 = 0";
+
+static char *update_unique_name =
+  "update _db_class set class_name = substring_index (class_name, '.', -1), unique_name = class_name where is_system_class % 8 = 0";
+
+static char *serial_query = {
+  "select \
+      'call change_serial_owner (''' || name || ''', ''' || substring_index (name, '.', 1) || ''') on class db_serial' as q \
+   from db_serial \
+   where class_name is not null and instr (name, '.') > 0"
+};
+
+static char *update_serial[] = {
+  "update db_serial set name = substring_index (name, '.', -1)",
+  "update db_serial set unique_name = lower (owner.name) || '.' || name"
+};
+
+static char *update_trigger = "update db_trigger set unique_name = lower (owner.name) || '.' || name";
+
+static char *index_query[] = {
+  "create unique index u_db_serial_name_owner ON db_serial (name, owner)",
+  "alter table db_serial drop constraint pk_db_serial_name",
+  "alter table db_serial add constraint pk_db_serial_unique_name primary key (unique_name)",
+
+  "alter table db_serial modify column name varchar not null",
+  "alter table db_serial modify column unique_name varchar not null",
+
+  "create index i__db_class_unique_name on _db_class (unique_name)",
+  "create index i__db_class_class_name_owner on _db_class (class_name, owner)",
+
+  "drop index i__db_class_class_name on _db_class"
+};
+
+static char *update_db_class = "update db_trigger set unique_name = lower (owner.name) || '.' || name";
```

Comment:
It is not a db_trigger.

Please change it as below.
```
update _db_class set unique_name = class_name where is_system_class % 8 != 0;
```

Reply:
Thanks, fixed.